### PR TITLE
refactor(ast_tools): only use type names to ignore fields in the `DeriveContentHash`.

### PR DESCRIPTION
**Pull request by rzvxa**
related to https://github.com/oxc-project/backlog/issues/107

### CODE REVIEW

**Comment by rzvxa** [comment]
* **#5573** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/5573?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rzvxa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: tasks/ast_tools/src/derives/content_hash.rs [hunk 1]
```diff
@@ -13,13 +13,11 @@ define_derive! {
...
 }
 
-const IGNORE_FIELDS: [(/* field name */ &str, /* field type */ &str); 6] = [
-    ("span", "Span"),
-    ("trailing_comma", "Span"),
-    ("this_span", "Span"),
-    ("scope_id", "ScopeId"),
-    ("symbol_id", "SymbolId"),
-    ("reference_id", "ReferenceId"),
+const IGNORE_FIELDS_TYPES: [/* type name */ &str; 4] = [
```
Comment (overlookmotel) [inline tasks/ast_tools/src/derives/content_hash.rs:16]: This is 100% nit, but I think `IGNORE_FIELD_TYPES` (singular "field") would be more grammatical.

**Review by overlookmotel — APPROVED** [review]

File: tasks/ast_tools/src/derives/content_hash.rs (hunk 1 above)
Comment (rzvxa) [reply tasks/ast_tools/src/derives/content_hash.rs:16]: Yes Sir.

Comment (overlookmotel) [reply tasks/ast_tools/src/derives/content_hash.rs]: Thanks Rez.

**Pull request merged**